apps/dashboard/components/multi-select.tsx

```diff
@@ -38,9 +37,13 @@ export const MultiSelect: React.FC<Props> = ({ options, placeholder, selected, s
   const handleUnselect = (o: Option) => {
     setSelected((prev) => prev.filter((s) => s.value !== o.value));
   };
-
+  const tempSelectables = options.filter((o) => !selected.includes(o));
+  const selectables = tempSelectables.filter((o) =>
```

Comment:
1. You could use `useMemo` instead of recalculating the filtered array at every render
2. You can also apply a more advanced filtering technique such as [match-sorter](https://www.npmjs.com/package/match-sorter) instead of ` o.label.toLowerCase().includes(inputValue.toLowerCase())` (only if the desired behavior is to custom-filter by more than one input and with different priorities)